Hunting Queries/SigninLogs/LoginSpikeWithIncreaseFailureRate.yaml

```diff
@@ -0,0 +1,62 @@
+id: 528c1708-a67e-4e2f-b76d-d5e5e88a22aa
+name: Anomalous spike in user logins with increase in signin failure rate to detect Credential Stuffing and other mass authentcation attacks.
```

Comment:
Can we shorten this name, this is what shows in the UX and generally we prefer 50 chars or less.  The description will help clarify further.

Reply:
Sorted, I didn't know this so will keep that in mind in future